app/lib/shared/urls.dart

```diff
@@ -189,3 +192,39 @@ String inferServiceProviderName(String url) {
   }
   return null;
 }
+
+/// The URLs provided by the package's pubspec or inferred from the homepage.
+class PackageLinks {
```

Comment:
Should this be placed in the model as part of a PackageVersion?

Reply:
Agreed, but in a later PR, because we have another `PackageLinks` object for pagination links, and I'd rather have that refactored/renamed/moved before making this as part of PackageVersion. Without the renaming there would be a few name collision.